lms/lmsdb/models.py

```diff
@@ -365,6 +365,40 @@ def to_choices(cls: enum.EnumMeta) -> Tuple[Tuple[str, str], ...]:
         return tuple((choice.name, choice.value) for choice in choices)
 
 
+class SolutionGradeMarkColors(enum.Enum):
+    BLUE = 'primary'
+    GRAY = 'secondary'
+    GREEN = 'success'
+    RED = 'danger'
+    YELLOW = 'warning'
+    AZURE = 'info'
+    LIGHT = 'light'
+    DARK = 'dark'
+
+    @classmethod
+    def to_choices(cls: enum.EnumMeta) -> Tuple[Tuple[str, str], ...]:
+        choices = cast(Iterable[enum.Enum], tuple(cls))
+        return tuple((choice.value, choice.name) for choice in choices)
+
```

Comment:
I think it's better to not have here a rigid structure.
Just in case someone will ditch bootstrap and won't have the same name conventions, or maybe [edit the sass](https://getbootstrap.com/docs/5.0/customize/color/) and add his own colors

Reply:
(Bonus: if there'll be a feature to just enter a color hex)